spec/integration/auto_encryption_spec.rb

```diff
@@ -4,6 +4,7 @@
 
 describe 'Auto Encryption' do
   require_libmongocrypt
+  min_server_version '4.2'
```

Comment:
We generally use `min_server_fcv` in preference to `min_server_version` since most times a server with a lower fcv doesn't work. I am not sure whether this matters for encryption.